appcenter-pre-build.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
Does this script get called automatically during the build process?

Reply:
Yep! Added comment